ci-operator/step-registry/hypershift/aws/create/hypershift-aws-create-chain.yaml

```diff
@@ -13,6 +13,9 @@ chain:
     - name: HYPERSHIFT_AWS_REGION
       default: "us-east-1"
       documentation: "The AWS region of the cluster."
+    - name: HYPERSHIFT_HC_ZONES
```

Comment:
Why don't we use `EXTRA_ARGS`?

Reply:
This param is added here so we don't have to pass unsupported flags through `EXTRA_ARGS`.  
Btw we already have an equivalent parameter called `HYPERSHIFT_ZONES` for `chain: hypershift-aws-setup-nested-management-cluster`.